.travis.yml

```diff
@@ -7,8 +7,13 @@ addons:
   chrome: stable
 before_script:
   - polymer lint
+  - >-
+    npm run update-types && git diff --exit-code || (echo -e
+    '\n\033[31mERROR:\033[0m Typings are stale. Please run "npm run
+    update-types".' && false)
 install:
   - npm install -g polymer-cli
+  - npm install
```

Comment:
Remove this? Travis automatically runs it?

Reply:
It doesn't automatically run this when it is in the `install` section

Reply:
Oh I see. Our Travis files seem inconsistent, some of the other ones don't use the install section at all.

https://github.com/PolymerElements/iron-behaviors/blob/master/.travis.yml

Oh well, fine.